frontend/src/components/Bar/Menu.tsx

```diff
@@ -21,7 +22,7 @@ const Menu = memo(function Menu({ onMenuItemClick }: MenuProps) {
     }
 
     return (
-        <Grid paddingY={2} container spacing={2} columns={{ xs: 1, lg: 2 }}>
+        <Masonry sx={{margin: 0}} spacing={2} columns={{ xs: 1, lg: 2 }}>
             {activeMenuItems.map((item) => (
                 <Grid key={item.id} item xs={1} lg={1}>
```

Comment:
You should remove these Grids as you removed the container Grid.

Reply:
Have fixed